.github/workflows/nightly_multibuild.yml

```diff
@@ -0,0 +1,115 @@
+name: nightly multibuild
+
+on:
+  push: null
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        platform: [x64]
+        PLAT: [i686, x86_64]
+        INTERFACE64: ['', '1']
+        MB_ML_VER: ['', 2010, 2014]
+        include:
+          - os: ubuntu-latest
+            PLAT: x86_64
+            INTERFACE64: '1'
+            MB_ML_LIBC: musllinux
+            MB_ML_VER: _1_1
+            platform: [x64]
+          - os: ubuntu-latest
+            PLAT: x86_64
+            INTERFACE64: ''
+            MB_ML_LIBC: musllinux
+            MB_ML_VER: _1_1
+            platform: [x64]
+    env:
+      REPO_DIR: OpenBLAS
+      MACOSX_DEPLOYMENT_TARGET: 10.9
+      MB_PYTHON_VERSION: ${{ matrix.python-version }}
+      TRAVIS_PYTHON_VERSION: ${{ matrix.python-version }}
+      MB_ML_LIBC: ${{ matrix.MB_ML_LIBC }}
+      MB_ML_VER: ${{ matrix.MB_ML_VER }}
+      INTERFACE64: ${{ matrix.INTERFACE64 }}
+      BUILD_DIR: ${{ github.workspace }}
+      SCIPY_WHEELS_NIGHTLY_ACCESS: ${{ secrets.SCIPY_WHEELS_NIGHTLY_ACCESS }}
+      MULTIBUILD_WHEELS_STAGING_ACCESS: ${{ secrets.MULTIBUILD_WHEELS_STAGING_BUILD }}
+      PLAT: ${{ matrix.PLAT }}
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+        fetch-depth: 0
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Set extra env
+      run: |
+        git ls-remote https://github.com/xianyi/OpenBLAS HEAD | sed -e "s/\s*HEAD$//" | xargs -I {} echo "OPENBLAS_COMMIT={}" >> $GITHUB_ENV;
```

Comment:
Couldn't you use `OPENBLAS_COMMIT=HEAD` or `OPENBLAS_COMMIT=develop` instead? Then rather than add a `NIGHTLY` env variable, you could check `$OPENBLAS_COMMIT == "HEAD"`.